core/domain/suggestion_services_test.py

```diff
@@ -1648,3 +1648,548 @@ def test_get_voiceover_application_from_model_with_invalid_type_raise_error(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        translation_suggestion = (
+            self._create_translation_suggestion_with_language_code(
+                self.language_code))
+        # Assert that the translation suggestion count increased.
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 1})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+        suggestion_services.reject_suggestion(
+            translation_suggestion.suggestion_id, self.reviewer_id,
+            'review message')
+
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 0})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+    def test_reject_translation_suggestions_same_lang_lowers_translation_count(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        translation_suggestion_1 = (
+            self._create_translation_suggestion_with_language_code(
+                self.language_code))
+        translation_suggestion_2 = (
+            self._create_translation_suggestion_with_language_code(
+                self.language_code))
+        # Assert that the translation suggestion count increased.
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 2})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+        suggestion_services.reject_suggestions(
+            [
+                translation_suggestion_1.suggestion_id,
+                translation_suggestion_2.suggestion_id
+            ], self.reviewer_id, 'review message')
+
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 0)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code,
+            {self.language_code: 0})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+    def test_reject_translation_suggestions_diff_lang_lowers_translation_count(
```

Comment:
I suggest also verifying that rejecting _one_ only reduces that one language (to guard against code that accidentally removes all counts from unintentionally leading to this test passing).

Reply:
Added :)